src/OrchardCore.Modules/OrchardCore.Contents/Controllers/AdminController.cs

```diff
@@ -176,37 +176,33 @@ public async Task<IActionResult> List(ListContentsViewModel model, PagerParamete
                 new SelectListItem() { Text = S["Delete"], Value = nameof(ContentsBulkAction.Remove) }
             };
 
-            // When using the AdminMenus ContentTypes Feature and specifying a Content Type hide the 'Content Type' filter.
```

Comment:
Ok, I tried this feature added by @PiemP and you are correct it doesn't work properly.

It was supposed to only hide the `ContentType` filter when using the `AdminMenus` `ContentTypes` menu listing (i.e. the one we have by default in the Blog recipe)

But as you say it is hiding them when selected from the normal contents listing (which is bad)

Is it possible to actually make the feature @PiemP wanted to add work, or are we getting rid of it, perhaps to be implemented correctly by @PiemP at a later stage.

Rest of pr looks fine

Reply:
Yeah I don't think this PR should fix this issue. I reverted the changes and made sure that the Content Type is selected in all cases instead. Should we prevent switching from one Content Type to an other when we display a specific list of Content Types? I'm not sure if it's necessary. The list now populates with Content Types that the user has authorization so there's no risks that it tries to list something that the user can't.

Reply:
I changed my mind, I'm going to fix it.

Reply:
Done